src/TestFramework/Coverage/JUnit/MemoizedTestFileDataProvider.php

```diff
@@ -33,11 +33,13 @@
 
 declare(strict_types=1);
 
-namespace Infection\TestFramework\Coverage\XmlReport;
+namespace Infection\TestFramework\Coverage\JUnit;
```

Comment:
AFAICS those changes of namespaces could be done in a separate PR for a bit of clarity

Reply:
Sure, #1131